roles/seed-git-server/README.md

```diff
@@ -0,0 +1,41 @@
+Seed Git Server
+====================
+
+Seeds a git server with example repositories 
+
+Requirements
+------------
+
+This role has dependencies that are not suitable for systems that are part of an OpenShift Cluster as it requires *firewalld* to be installed
```

Comment:
Maybe update this "requirement" to remove the mention of firewalld as that's for the git-server role rather than this role. However, maybe just list a requirement that a git server needs to be operational (in addition to the `Dependencies` below).